### PR TITLE
[FIX] keyboard moves view when inputs are focused

### PR DESCRIPTION
**Pull request by ericgibb**

### CODE REVIEW

**Comment by ericgibb** [comment]
fixes #263 
fixes #248 
fixes #246

File: src/release_notes.json [hunk 1]
```diff
@@ -1,4 +1,12 @@
 [
+  {
+    "release_notes": "Mainnet build\nFixed Pin Input Screens styles\nFixed bug persisting old network transactions\nFixed recovery screen styles, next buttons",
+    "version_code": "1.12"
+  },
+  {
+    "release_notes": "Mainnet build\nFixed inconsistent Pin Input Screens\nFixed auto-selecting next input field\nFixed keyboard covering inputs",
+    "version_code": "1.12"
+  },
```
Comment (zachgibb) [inline src/release_notes.json]: ✂️ duplicate

File: src/screens/Settings/Profile/Profile.js [hunk 2]
```diff
@@ -147,6 +147,9 @@ export default class Profile extends Component {
   };
 
   safeFocus = memoize(element => () => {
+    if (!element) {
```
Comment (zachgibb) [inline src/screens/Settings/Profile/Profile.js]: when / how does this happen?

File: src/screens/Wallet/components/InputList.js [hunk 3]
```diff
@@ -55,41 +58,59 @@ export default class InputList extends Component {
...
     this.props.saveAndContinue();
   };
 
   render() {
     const { answers, offset, onCancel } = this.props;
     const { completed } = this.state;
 
+    const RENDER_INPUTS = [];
+
+    answers.forEach((answer, i) => {
```
Comment (zachgibb) [inline src/screens/Wallet/components/InputList.js]: is pulling this out of the JSX block intended to fix the issue where rerenders of the children will happen for each render of this parent component? if so, having this `forEach` in the `render` function only kinda shifts things around - not actually saving renders.

**Comment by ericgibb** [comment]
fixes #268

**Pull request merged**